inventory

```diff
@@ -16,6 +16,9 @@ template_service_broker_install=false
 openshift_use_manageiq=false
 openshift_install_examples=false
 
+# Enabling KubeVirt Admission Controller
```

Comment:
nit: why not add docs ref like cnv repo did ?
```
# Following option is here becasue of:
# https://www.kubevirt.io/user-guide/#/installation/api-validation?id=kubevirt-api-validation
```

Reply:
:+1: